cairo_nox/meta.yaml

```diff
@@ -1,6 +1,6 @@
 # adapted from: https://github.com/conda/conda-recipes/blob/master/cairo/meta.yaml
 package:
-  name: cairo
+  name: cairo_nox
```

Comment:
is this renaming back to `cairo_nox` intended?

Reply:
yeah, I thought it was worth making it very clear that it's different (in some way at least) from the other cairo recipes.

Reply:
but the different naming prevents the regular and nox variants of cairo from being used interchangeably with rdkit (that I think was the original intention for the nox feature metapackage), or am I confusing things?

Reply:
That's a valid point, and one I forgot. Let me re-think that